lib/index.js

```diff
@@ -351,14 +356,13 @@ module.exports = function(options) {
     logMessage(options.debug, 'addTxIdToResponse took time ', timeTookInSeconds(addTxIdToResponseStartTime, addTxIdToResponseEndTime));
 
     res.end = function(chunk, encoding, callback) {
-      logMessage(options.debug, 'response end', 'append chunk=' + chunk);
-
       var finalBuf = resBodyBuf;
 
       res.time = new Date();
       res.responseTime = new Date() - req._startTime;
 
-      if (chunk && typeof chunk !== 'function') {
+      if (chunk && typeof chunk !== 'function' && options.logBody && (chunk.length < options.responseMaxBodySizeLimit)) {
```

Comment:
i dont know how chunk works here.. So lets say first chunk is < sizeLimit, but chunk1 + chunk2 > sizeLimit, is it taken care of here?

Reply:
chunk would be an entire response and we'd use this https://github.com/Moesif/moesif-nodejs/blob/master/lib/index.js#L481 to append chunk to our local variable `finalBuf` (buffer). Based on all the testing I did, I've never came across a case where there would be chunk1 + chunk2, so with chunk < sizeLimit all the cases will be covered. 
@xinghengwang let us know if we got it correct and/or is there any case where there would be chunk1+ chunk2?

Reply:
@xinghengwang  can you review this pr when you get time? Thanks

Reply:
Chatting with Xing here, we do believe there are cases where we have multiple chunks (which is why there is the appendChunk) so agree we should accumulate. It may be dependent on how response is sent (chunked or not) or other node configuration